samples/http_2.js

```diff
@@ -0,0 +1,7 @@
+import http from "k6/http";
+import { check } from 'k6';
+
+export default function() {
+  const HOMEPAGE_RESPONSE = http.get("https://www.bbc.co.uk/");
+  check(HOMEPAGE_RESPONSE, { "HTTP 2.0": (r) => r.proto  === "HTTP/2.0" });
```

Comment:
The formatting used in the rest of the samples is:

```js
check(http.get("https://www.bbc.co.uk/"), {
    "status is 200": (r) => r.status == 200,
    "protocol is HTTP/2": (r) => r.proto == "HTTP/2.0",
});
```

Reply:
Also: are you sure this is correct? I seem to remember the `proto` value for HTTP/2 being `"h2"`, but I could be wrong.

Reply:
https://github.com/loadimpact/k6/blob/80c201125846f9954312e8f65cc40111b04c5fc4/js/modules/k6/http/http_test.go#L219